Lib/glyphsLib/builder.py

```diff
@@ -344,10 +344,6 @@ def set_custom_params(ufo, parsed=None, data=None, misc_keys=(), non_info=()):
         if name.startswith('openTypeOS2Win') and value < 0:
             value = -value
 
-        # Some Glyphs files store weightClass/widthClass as string values.
-        if name in ('openTypeOS2WeightClass', 'openTypeOS2WidthClass'):
-            value = int(value)
```

Comment:
I think we should still keep this `value = int(value)`, as the value could be a float, and ufoLib/defcon expect an int.